### PR TITLE
Upgrade to Angular 2.1.1

### PR DESCRIPTION
**Pull request by harrybabu**
1) Updated Angular2 to 2.1.1 
2) Updated few other dependcies of typescript and loader
3) Removed transfering webpack sourcemap for production mode
4) Added preload modules to test angular2.1.1 ( Will be implementing lazy loading of routes)

### CODE REVIEW

File: config/webpack.prod.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const METADATA = webpackMerge(commonConfig.metadata, {
 });
 
 module.exports = webpackMerge(commonConfig, {
-  devtool: 'source-map',
+  // devtool: 'source-map',
```
Comment (harrybabu) [inline config/webpack.prod.js]: okies will do that

File: package.json [hunk 2]
```diff
@@ -19,6 +19,10 @@
     {
       "name": "Sudipta Sen",
       "email": "sudsen@redhat.com"
+    },
+    {
+      "name": "Haribabu Viswanathan",
```
Comment (harrybabu) [inline package.json]: okies will do that

**Comment by ppitonak** [comment]
@joshuawilson can you start the Jenkins job? Just comment on this PR with `ok to test`, alternatively whitelist @harrybabu with comment `add to whitelist`

**Comment by joshuawilson** [comment]
@harrybabu add to whitelist

**Comment by joshuawilson** [comment]
@harrybabu upgrading the version of Angular is not simply a change of the version numbers. The Angular team continues to change the API. It takes a careful review of the Angular docs and of our own code to apply those changes. I had started that review before you made this commit. I have continued with this review. That is why I assigned the issue to myself. I am almost done with the update. I'm going to close this PR and hopefully submit a new PR for this issue very soon.

**Pull request closed without merging**